TBG.Business/BusinessController.cs

```diff
@@ -9,5 +9,38 @@ namespace TBG.Business
 {
     public class BusinessController : IController
     {
+        public void TestTournament()
+        {
+            var tournament = new SingleEliminationTournament()
+            {
+                TournamentId = 1,
+                TournamentName = "Test Tournament",
+                EntryFee = 50,
+                TotalPrizePool = 2000,
+                TournamentTypeId = 1,
+                UserId = 1
+            };
+
+            for (int i = 1; i <= 8; i++)
+            {
+                var te = new TournamentEntry()
+                {
+                    TournamentEntryId = i,
+                    TournamentId = 1,
+                    TeamId = i,
+                    Seed = i
+                };
+
+                tournament.Participants.Add(te);
+            }
+
+            tournament.BuildTournament();
+            var matchup = tournament.Rounds.Find(x => x.RoundNum == 1).Pairings.Find(x => x.MatchupId == 1);
```

Comment:
It's not very obvious to me what this does. Can you explain?

Reply:
.Find is a Linq method on a List.  Linq uses a query structure that looks like this x => x.something
This line of code is calling Find on the Rounds list, to find the first round.  It then calls Find on the Pairings list, to find the first matchup.

This is part of the "throw away" code.  I'm just manually pulling a matchup and settings the scores, which will be done from the GUI instead of this method.